### ISSUE TITLE
Crypto donations

### ISSUE THREAD

**Issue by IntinteDAO** [Jmgr/actiona#172]
Hello, maybe there will be a sponsorship option using cryptocurrencies? Admittedly I won't transfer much myself, but I believe it would be a step forward.

**Comment by Jmgr** [comment]
Hi. Thanks ! I would prefer regular sponsorship if possible due to the environmental impact of most cryptocurrencies, but if you have no other choice then I can have a look.

**Comment by IntinteDAO** [comment]
Of course, you can choose cryptocurrencies that do not consume a lot of energy. On the other hand, I think it's a good idea to have a couple of wallets and whoever wants - subsidize.

**Comment by Jmgr** [comment]
I can now receive Cardano (ADA): addr1v9zj0klgp8pwa6kf7956nfj5qn7gjuw5x2lyt86vhcwj0xg0eytad -- thanks!

**Comment by IntinteDAO** [comment]
Ok, sent. But you should add crypto addresses to README.md I think :)

**Comment by Jmgr** [comment]
Received, much appreciated!

> But you should add crypto addresses to README.md I think :)

Good idea.

> Go go Actiona!

There are a few challenges coming, like Wayland, Qt 6, MacOS and tackling the long list of wanted features, but we'll get there eventually. People showing interest definitely helps :smiley:
